app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -2031,6 +2032,21 @@ class SectionRenderingService(
 
     val isPageHeading = ei.formLevelHeading
 
+    val singleton = ei.singleton
+    val page = singleton.page
+
+    val formComponents = ei.formModelOptics.formModelVisibilityOptics.formModel(ei.sectionNumber).allFormComponents
+
+    val tableComps = formComponents.collect { case IsTableComp(tc) => tc }
+    val miniSummaryListComps = formComponents.collect { case IsMiniSummaryList(tc) => tc }
```

Comment:
If this method is rendering file upload component, why is here logic for table component or mini summary page?